_posts/fr/2018-08-31-ddp.md

```diff
@@ -0,0 +1,130 @@
+
+Il y a six volets de travail distincts pour lesquels vous pouvez soumettre une proposition. Vous pouvez soumettre des propositions pour un ou plusieurs de ces volets ou sous-volets :
+
+
+
+1.  **[Mise en œuvre du gouvernement numérique](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=74)**
+
+1.  **[Recrutement](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=75)**
+
+    2.1 **[Talents](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=76)**
+
+
+    2.2 **[Recrutement d'utilisateurs](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=77)**
+
+1.  **[Formation](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=77)**
+
+3.1 **[La conception centrée sur l'utilisateur](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=78)**; 
+
+3.2 **[Le développement de produits de façon agile](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=78)**; ou,
+
+3.3 **[La littératie numérique des décideurs](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=78)**.
+
+*Nous aurons des opportunités futures pour les entreprises de prestation de services qui s'intéressent à travailler avec le SNC sur la conception et le développement de produits.
+
+
+### 3. Choisissez ce pour quoi vous soumettez une proposition
+
+Envisagez les volets et sous-volets d'application comme des choix de menu à la carte. Vous pouvez choisir de soumettre une proposition pour une, plusieurs ou toutes les options. Vous êtes tenu de présenter une proposition seulement pour le ou les (sous)volets d'application que vous avez choisis. 
+
+Par exemple, si vous vous spécialisez dans le développement et la formation agiles, vous pourriez soumettre une proposition pour le sous-volet 3.2. Préparez une soumission qui englobe autant (ou aussi peu) de champs que vous le souhaiterez. 
+
+
+### 4. Nous prévoyons attribuer plusieurs contrats
+
+Pour chacun des champs d'application, nous prévoyons attribuer plusieurs contrats en fonction des [critères d'évaluation](https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=32). Les valeurs combinées maximales des contrats attribuables à chacun des volets et sous-volets se retrouvent dans la DP. Au total, nous prévoyons attribuer [jusqu'à 17 contrats](https://buyandsell.gc.ca/cds/public/2018/08/31/5f22b5c7f0a01471c25d49629bf1c584/19-130_rfp_eng.pdf#page=26) pour une valeur totale de 1 175 000 $. 
```

Comment:
There’s a link to the English RFP here, the second link.

Reply:
It should be a link to the French one: https://buyandsell.gc.ca/cds/public/2018/08/31/7255f4e7b9163cdbe61fa80aba9546fa/19-130_rfp_fra.pdf#page=29